crates/bevy_ecs/src/schedule/system_descriptor.rs

```diff
@@ -45,6 +45,12 @@ pub trait IntoSystemDescriptor<Params> {
 
 pub struct SystemLabelMarker;
 
+impl IntoSystemDescriptor<()> for SystemDescriptor {
+    fn into_descriptor(self) -> SystemDescriptor {
+        self
+    }
+}
```

Comment:
What does this do?

Reply:
It allows passing a `SystemDescriptor` for methods that accept `IntoSystemDescriptor`. It's similar to `impl From<T> for T`.

Reply:
Could you add this as a quick doc comment?

Reply:
> Could you add this as a quick doc comment?

Not necessary, in my opinion, the trait name is clear enough. I was confused because I thought we already had the implementations we needed, and we do: this one just wasn't necessary. It is now.